### ISSUE TITLE
No module name zato.common.api

### ISSUE THREAD

**Issue by efgieha** [zatosource/zato#1088]
Hi @dsuch! I downloaded your latest changes from zato-client and unfortunately I've got no module error. I think that you missing something  when you moving zato-common to zato-cy (https://github.com/zatosource/zato/commit/339c848b52689c9c31ebdce2acee5bc6c34c0e55)
![image](https://user-images.githubusercontent.com/5695085/96414856-f4634580-11ed-11eb-9191-4f0f2415eb55.png)

**Comment by DimShadoWWW** [comment]
Hello @efgieha ,

Please, can you try to do a full update following the instructions in https://zato.io/docs/admin/guide/install/update.html and check again if the error persists ?

Thanks

**Comment by efgieha** [comment]
But this is not error inside zato-server (btw I have docker version of zato and I saw that update.sh start on every build). 
This problem exists when:
- I install zato-client from repo inside my app (pip install -e git+https://github.com/zatosource/zato.git#"egg=zato-client&subdirectory=code/zato-client"). Zato-client install the latest version of zato-common that didn't work. So I check what change in zato-common and I saw that you move zato-common api code to zato-cy. 
I install zato-client and zato-common from commit before this migration and it works properly (commitID: 0e9f14f93dce1cee0e4ff5cfe4169ec0e412835a)
